### PR TITLE
Add proxy support article

### PR DESCRIPTION
**Pull request by zax71**

### CODE REVIEW

**Comment by zax71** [comment]
I didn’t mention Bungee here, I presume it is supported but I’ve never done it myself. If someone could chime in on that then that would be helpful

File: docs/feature/proxy-support.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+It is as simple as adding the below between `MinecraftServer.init()` and `MinecraftServer#start`
+
+```java
+String secret = "My really secret secret that is not hard coded"
+VelocityProxy.enable(secret)
+```
+
+## Transferring between servers
+
+You need to install [Guva](https://github.com/google/guava) to transfer players. Once doing that, just add the below to send the appropriate packet
```
Comment (mworzala) [inline docs/feature/proxy-support.md:20]: You do not need guava to create this buffer. You can use the `NetworkBuffer` api:

Also should link to https://docs.papermc.io/paper/dev/plugin-messaging#what-did-we-just-do somewhere.

```
        p.sendPluginMessage("BungeeCord", NetworkBuffer.makeArray(buffer -> {
            buffer.write(NetworkBuffer.STRING, "Connect");
            buffer.write(NetworkBuffer.STRING, "lobby");
        }));
```

**Review by mworzala — CHANGES_REQUESTED** [review]

**Comment by mworzala** [comment]
> I didn’t mention Bungee here, I presume it is supported but I’ve never done it myself. If someone could chime in on that then that would be helpful

Bungee is supported via `BungeeCordProxy.enable()`, and BungeeGuard is supported by adding `BungeeCordProxy.setBungeeGuardTokens(Set.of("tokens", "here"))`

**Comment by zax71** [comment]
Thanks for the feedback! I’m on holiday currently so it’ll probably be a week or two before I can address these.

**Comment by mworzala** [comment]
No problem, have a nice holiday

**Review by mworzala — APPROVED** [review]

**Pull request merged**